### PR TITLE
Custom PR description file

### PR DESCRIPTION
**Pull request by Dan7-7-7**
This addresses https://github.com/Skyscanner/turbolift/issues/95 and https://github.com/Skyscanner/turbolift/issues/73. It adds the option to take the PR title and description from a different file to the default README.md, using a flag `--description` for create-prs.

I've also slightly modified the default README.md as suggested in https://github.com/Skyscanner/turbolift/issues/27.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,12 +172,14 @@ Use `turbolift create-prs --sleep 30s` to, for example, force a 30s pause betwee
 > * create PRs in batches, for example by commenting out repositories in `repos.txt`
 > * Use the `--draft` flag to create the PRs as Draft
 
-If you need to mass-close PRs, it is easy to do using `turbolift foreach` and the `gh` GitHub CLI ([docs](https://cli.github.com/manual/gh_pr_close)):
```
Comment (Dan7-7-7) [inline README.md:175]: It looks like this line is superseded by the new update-prs --close command which is explained below

File: internal/campaign/campaign.go [hunk 2]
```diff
@@ -131,10 +135,13 @@ func readReposTxtFile(filename string) ([]Repo, error) {
 	return repos, nil
 }
 
-func readPrDescriptionFile() (string, string, error) {
-	file, err := os.Open("README.md")
+func readPrDescriptionFile(filename string) (string, string, error) {
+	if filename == "" {
```
Comment (Dan7-7-7) [inline internal/campaign/campaign.go:139]: I have copied this check from the --repos flag, but I'm wondering whether it is really needed (won't it error if the flag has no argument?)

File: internal/campaign/campaign_test.go [hunk 3]
```diff
@@ -218,7 +218,39 @@ func TestItShouldAcceptADifferentRepoFileNotExist(t *testing.T) {
 func TestItShouldErrorWhenRepoFileIsEmpty(t *testing.T) {
 	testsupport.PrepareTempCampaign(false)
 
-	options := &CampaignOptions{}
+	options := NewCampaignOptions()
+	options.RepoFilename = ""
```
Comment (Dan7-7-7) [inline internal/campaign/campaign_test.go:222]: changed this around in order to test each value separately

**Comment by arnaugm** [comment]
Awesome!

**Pull request merged**